code/modules/mob/living/carbon/damage_procs.dm

```diff
@@ -84,9 +84,9 @@
 	if(!forced && HAS_TRAIT(src, TRAIT_TOXINLOVER)) //damage becomes healing and healing becomes damage
 		amount = -amount
 		if(amount > 0)
-			blood_volume -= 5*amount
+			blood_volume = max(blood_volume - (5*amount), 0)
 		else
-			blood_volume -= amount
+			blood_volume = max(blood_volume - amount, 0)
```

Comment:
Wouldn't max 0 in this case cause you to become stuck at 0% blood if they somehow become negative and then attempt to restore it with something like synthflesh?

Reply:
Max takes the largest of the two values, so it sets `blood_volume` to `(blood_volume - amount)` or `0`, whichever his higher.

Reply:
Hmm, unless I missed something, I don't think I spotted any other mechanism that would allow their blood to go negative.

Even if it did, it should still immediately jump back up to 0, as all negative numbers would be lower than 0.

Reply:
That's the thing, it only goes to the else if `adjustToxLoss` is called with a positive value, which is then inverted.
`blood_volume` - `amount` in this case  will always be a positive value, unless someone creates a situation in which `blood_volume` is negative. 
If the result is a positive value max 0 does nothing, and so that check is only preventing people from treating those with negative `blood_volume` through knowledge of systems available.

Reply:
I think I'm having brain worms.

Reply:
Doesn't really matter either way, I think. No matter if it's positive or negative, if any attempt it made to set blood volume lower than 0, it'll re set to 0.